### ISSUE TITLE
vertices_latitude / vertices_longitude should not have attribute units

### ISSUE THREAD

**Issue by larsbuntemeyer** [WCRP-CORDEX/cordex-cmip6-cmor-tables#68]
I am not sure why this is the case for the [CMIP6_grids.json](https://github.com/PCMDI/cmip6-cmor-tables/blob/main/Tables/CMIP6_grids.json) table but i think we should not have the units attribute in the boundary variables according to [CF conventions](https://cfconventions.org/cf-conventions/cf-conventions.html#cell-boundaries):

https://github.com/WCRP-CORDEX/cordex-cmip6-cmor-tables/blob/80f7f510266884d372d68de6f4d63ffd68255f18/Tables/CORDEX-CMIP6_grids.json#L137-L156

**Comment by gnikulin** [comment]
I agree, the units attribute in the boundary variables is redundant information and the boundary variables shouldn't have any attributes.

**Comment by larsbuntemeyer** [comment]
OK, we can't really do anything about what cmor will write out here i guess. But the way it is now should be fine since these are recommendations. I'll leave the issue open anyway..
